src/components/ContactOption.jsx

```diff
@@ -17,7 +17,9 @@ function ContactOption(props) {
       .then((response) => response.json())
       .then((data) => {
         setContacts(data);
-        setDefaultValue(data[0].id);
+        if (data !== null) {
```

Comment:
You should be checking
```js
if (data.length !== 0)
```

Reply:
This makes more sense. Thanks! I updated